README.md

```diff
@@ -113,27 +113,59 @@ that are loaded when importing Python packages:
   'version': None}]
 ```
 
-In the above example, `numpy` was installed from the default anaconda channel and
-comes with the MKL and its Intel OpenMP (`libiomp5`) implementation while
-`xgboost` was installed from pypi.org and links against GNU OpenMP (`libgomp`)
-so both OpenMP runtimes are loaded in the same Python program.
+In the above example, `numpy` was installed from the default anaconda channel and comes
+with MKL and its Intel OpenMP (`libiomp5`) implementation while `xgboost` was installed
+from pypi.org and links against GNU OpenMP (`libgomp`) so both OpenMP runtimes are
+loaded in the same Python program.
+
+The state of these libraries is also accessible through the object oriented API:
+
+```python
+>>> from threadpoolctl import threadpool_info
+>>> from pprint import pprint
+>>> import numpy
+>>> controller = ThreadpoolController()
+>>> pprint(controller.info())
```

Comment:
Here I would just use the imported function.

```suggestion
>>> pprint(threadpool_info())
```

Reply:
I meant to import the class. There's an example with the function just above and this snippet is here to describe the object oriented API.